website/content/en/preview/faq.md

```diff
@@ -112,11 +112,20 @@ If the instance type is unavailable for some reason, then fleet will move on to
 If you are using the spot capacity type, Karpenter uses the capacity-optimized-prioritized allocation strategy which tells fleet to find the instance type that EC2 has the most capacity of which will decrease the probability of a spot interruption happening in the near term.
 See [Choose the appropriate allocation strategy](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-fleet-allocation-strategy.html#ec2-fleet-allocation-use-cases) for information on fleet optimization.
 
-### What if there is no spot capacity? Will Karpenter use on-demand?
+### What if there is no Spot capacity? Will Karpenter use On-Demand?
 
-Karpenter will use on-demand, if your provisioner specifies both spot and on-demand and you are flexible to at least 5 instance types. 
+The best defense against running out of Spot capacity is to allow Karpenter to provision as many different instance types as possible.
+Even instance types that have higher specs, e.g. vCPU, memory, etc., than what you need can still be cheaper in the Spot market than using On-Demand instances.
+When Spot capacity is constrained, On-Demand capacity can also be constrained since Spot is fundamentally spare On-Demand capacity.
+Allowing Karpenter to provision nodes from a large, diverse set of instance types will help you to stay on Spot longer and lower your costs due to Spot’s discounted pricing.
```

Comment:
Do we want to mention that this is because we use `capacity-optimized` for our EC2 fleet calls to pick the deepest pools?

Reply:
CO is more about interruption rate, not availability.